src/Components/FullNavBar.js

```diff
@@ -18,15 +26,15 @@ class FullNavBar extends Component {
           </Navbar.Brand>
         </Navbar.Header>
         <Navbar.Collapse>
-          <Nav>
-            <NavItem eventKey={1} href="#">
-              Link
+          <Nav onSelect={this.handleSelect}>
+            <NavItem eventKey={this.complaintsRoute}>
+              Complaints
             </NavItem>
-            <NavItem eventKey={2} href="#">
+            <NavItem eventKey={2}>
               Link
```

Comment:
Esto no hace nada por ahora no?

Reply:
no aum. its gonna be legend.. wait for it ...